pkgs/by-name/lu/luci-go/package.nix

```
@@ -0,0 +1,46 @@
+{
+  buildGoModule,
+  fetchgit,
+  lib,
+}:
+let
+  commit = "500493c154652d6986a34b341e98df244ae1ad0d";
+in
+buildGoModule rec {
+  pname = "luci-go";
+  version = "0-unstable-2024-10-31";
+
+  src = fetchgit {
+    url = "https://chromium.googlesource.com/infra/luci/luci-go";
+    rev = commit;
+    hash = "sha256-HP4Aizt5FJA3IAlqs7gylw8/xUbBwsmReGaR8jIkmrk=";
+  };
+
+  vendorHash = "sha256-FMqbEls6MivPeReZTADrfcAvxo8o0Gy7bq9xG6WN38k=";
+
+  checkFlags =
+    let
+      skippedTests = [
+        # require network access
+        "TestDownloadInputs"
+        "TestInstallCipd"
+        "TestIsLocalAddr"
+        "TestGenerateSignedURL"
+
+        # require filesystem access
+        "TestPythonBasic"
+        "TestPythonFromPath"
+      ];
+    in
+    [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
+
+  meta = with lib; {
+    description = ''
+      LUCI services and tools in Go which is part of Chromium infra
+    '';
```

```suggestion
    description = ''LUCI services and tools in Go which is part of Chromium infra'';
```

---

Thanks. What's the advantage? Don't really like over-long lines.

---

Because description is expected to be one line. See https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

The above suggestion is not accurate:

```suggestion
    description = "LUCI services and tools in Go which is part of Chromium infra";
```

---

Ok. I was thinking the one line restriction refers to the evaluated string, not its source expression.

Anyway, I changed the metadata.